lambdas/process_webhooks/process_webhooks.py

```diff
@@ -21,20 +21,36 @@
 logger.setLevel(numeric_level)
 
 
-def _get_target_url():
+def _get_target_url(headers):
     """
-    Get the target URL for the processed hooks from a comma delimited
-    list of URL, set in an OS environment variable.
+    Get the target URL for the processed hooks from the
+    OS environment variable. Based on the GitHub event,
+    add the proper endpoint.
 
-    Return the target URL
+    Return the target URL with the appropriate endpoint
     """
     url = os.environ.get('TARGET_URL')
     if not url:
         raise StandardError(
             "Environment variable TARGET_URL was not set"
         )
 
-    return url
+    event_type = headers.get('X-GitHub-Event')
+
+    # Based on the X-Github-Event header, determine the
+    # proper endpoint for the target url.
+    # PR's and Issue Comments use the ghprb Jenkins plugin
+    # Pushes use the Github Jenkins plugin
+    if event_type in ["issue_comment", "pull_request"]:
+        endpoint = "ghprbhook/"
+    elif event_type == "push":
+        endpoint = "github-webhook/"
+    else:
+        raise StandardError(
+            "This webhook has an unsupported GitHub event type."
+        )
+
+    return url + "/" + endpoint
```

Comment:
nit: we will need to be careful when setting the base URL to NOT include a "/" at the end.
Which I think is OK for now since we're going to deprecate setting it as an environment variable. Something to keep in mind tho.